no_drama/__main__.py

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/env python
+
+import argparse
+import tempfile
+import os
+import os.path
+import shutil
+import subprocess
+import zipfile
+import glob
+import fnmatch
+
+from string import Template
+from functools import partial
+
+
+this_file = os.path.realpath(__file__)
+this_directory = os.path.dirname(this_file)
+templates_dir = os.path.join(this_directory, 'templates')
+build_lib = os.path.join(this_directory, 'build_lib')
+build_files = os.path.join(this_directory, 'build_files')
+
+
+def save_wheels(destination, packages=None, requirements_file=None,
+                find_links=None):
+    call_list = ["pip", "wheel", "--wheel-dir=%s" % destination]
+
+    if packages is not None:
```

Comment:
in this case, i think we expect packages to be a list.  so you don't really have to worry about the boolean value of False here.

In fact, if you do pass in a boolean, the above code will still fail.

You can't use the += operator with a list and a boolean.

That operator tells me packages HAS to be a list.